frontend/vue/components/CodeExercise/CodeOutput.vue

```diff
@@ -62,12 +66,15 @@ export default defineComponent({
           })
         } catch (error: any) {
           this.error = error as string
-          this.outputArea!.model.clear()
+          this.outputArea!.setHidden(false)
         }
       })
     },
     setOutputFuture (future : IOutputShellFuture) {
       this.outputArea!.future = future
+    },
+    clearOutput () {
+      this.outputArea!.setHidden(true)
```

Comment:
quick question.... maybe i am misunderstanding but why hide instead of clear?

Reply:
@vabarbosa - for some reason, the clear wasn't working... @korgan00 and I were pairing trying to figure it out but it wasn't clearing 🤔

Reply:
[Here's an issue](https://github.com/Qiskit/platypus/issues/1328) so we can investigate this more.